### PR TITLE
Error handling (Kafka Streams)  and Filtering (ksqlDB and Kafka Streams)

### PR DESCRIPTION
**Pull request by bbejeck**

### CODE REVIEW

File: filtering/ksql/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Filtering
+
+How do you filter messages in a Kafka topic to contain only those that you're interested in?
+
+
+## Setup
+
+First, you'll create a stream based on the topic you're interested in only seeing a sub-set of the records.
```
Comment (davetroiano) [inline filtering/ksql/README.md:8]:
```suggestion
First, you'll create a stream based on the topic you're interested in only seeing a subset of the records.
```

File: filtering/ksql/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+In this case, it's a topic containing with events representing book publications. You want a query that creates a new topic containing only events for a particular author.
+
+```sql
+CREATE STREAM george_martin WITH (kafka_topic = 'george_martin_books') AS
+    SELECT *
+      FROM all_publications
+      WHERE author = 'George R. R. Martin';
+```
+
+Now you have a new stream that will place the filtered results in a new topic
```
Comment (davetroiano) [inline filtering/ksql/README.md:26]:
```suggestion
Now you have a new stream that will place the filtered results in a new topic.
```

**Review by davetroiano — APPROVED** [review]
LGTM!

**Comment by bbejeck** [comment]
> [bbejeck](https://github.com/bbejeck) dismissed [davetroiano](https://github.com/davetroiano)’s [stale review](https://github.com/confluentinc/tutorials/pull/28#pullrequestreview-1841725603) via [24d16f8](https://github.com/confluentinc/tutorials/commit/24d16f826584c14b6570850fc4a4ee93cbd83f9b)
No GitHub I didn't dismiss @davetroiano's review - you have a problem with us making suggested changes to an approved PR

**Review by davetroiano — APPROVED** [review]
LGTM

**Pull request merged**